### PR TITLE
Reference Greg Version 2.5.0.5076 which has a dotnet 6 target

### PR DESCRIPTION
**Pull request by Cristhian-Marin-asdk**
### Purpose

Upgrade Greg Client package to 2.5.0.5076 which now has a dotnet 6 target

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB 

### Release Notes

Upgrade Greg Version to 2.5.0.5076

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.19.0.4737")]
+[assembly: AssemblyFileVersion("2.19.0.5077")]
```
Comment (mjkkirschner) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:67]: please don't commit this file in general - it just creates conflicts for others prs.

**Review by mjkkirschner — APPROVED** [review]
1. lets wait for the tests
2. one change request 
otherwise LGTM

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs (hunk 1 above)
Comment (Cristhian-Marin-asdk) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:67]: Ok, I'll avoid commiting this file next time

**Comment by JimmySanchezGlobant** [comment]
LGTM

**Review by JimmySanchezGlobant — APPROVED** [review]
LGTM

**Pull request merged**